client/deviceauth/client_deviceauth.go

```diff
@@ -88,11 +91,20 @@ func (d *Client) UpdateDevice(ctx context.Context, sreq StatusReq) error {
 	}
 	defer rsp.Body.Close()
 
-	if rsp.StatusCode != http.StatusNoContent {
+	switch rsp.StatusCode {
+	case http.StatusNoContent:
+		return nil
+	case http.StatusUnprocessableEntity:
+		ae, err := utils.ParseApiErr(rsp)
+		if err == nil {
+			return &utils.UsageError{UserMsg: ae.Error}
```

Comment:
Think we should add a helper for generating usage errors, something like `NewUsageErr(message string) error`